### DISCUSSION TITLE
[Request] Add warning for tick lag

### DISCUSSION THREAD

**Discussion by redmoogle** [ldtteam/minecolonies#9014] [Ideas]
I was having an issue where my update rate was far too fast to keep up and it kept breaking it. Might be a good idea imo to warn users if the updates cant keep up

**Comment by MotionlessTrain** [comment]
Please use https://github.com/ldtteam/minecolonies-features for feature requests.
And I'm not sure whether mods can detect the tick rate at all

**Comment by Raycoms** [comment]
I'd like to first see spark profiles of the lag

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by redmoogle** [comment]
[sparkprofiles.zip](https://github.com/ldtteam/minecolonies/files/11007158/sparkprofiles.zip)

with 10 updates per some unit of time it takes about 20 seconds for the builder to decide to do something

**Comment by redmoogle** [comment]
![image](https://user-images.githubusercontent.com/20369082/226075449-e89fd4d5-d5a2-40b7-9e86-8dcfeb229847.png)
well I somehow managed to read that inversely, however 10 throttling it down to nothing seems a bit... aggressive.

**Reply by uecasm** [reply]
1 is normal speed.  10 is 1/10 of normal speed (10% speed).

**Comment by Talyda** [comment]
Spark should have given you links to share for the profile, so that people don't have to download some random file from you I thought?

**Reply by redmoogle** [reply]
> Spark should have given you links to share for the profile, so that people don't have to download some random file from you I thought?

didnt know if links were permanent so I just grabbed the sparkfiles; although I just realized I read the comment wrong and increasing the value slows things down

**Reply by uecasm** [reply]
The default settings have them processing fast, although some modpacks slow them down by default.

**Reply by Raycoms** [reply]
Please give us a spark link (should be able to find tha tin your browser history)

**Comment by redmoogle** [comment]
redid the tests for accuracy

config value of 2 for updateRate
https://spark.lucko.me/MKfNCMXd1H

config value of 10 for updateRate (builder didnt even say they started building)
https://spark.lucko.me/YV0TN0ApGd

**Comment by Raycoms** [comment]
I dont see the problem. Your server is mostly idle.

**Reply by redmoogle** [reply]
> I dont see the problem. Your server is mostly idle.

yes i managed to read it backwards and thought increasing made it faster

**Reply by Raycoms** [reply]
Ah okay
